sample-docs/mkdocs.yml

```diff
@@ -3,6 +3,7 @@ site_name: Cats API
 nav:
   - Intro: 'index.md'
   - Authentication: 'authentication.md'
+  - '*include ./components/*/ Components': '!include mkdocs.yml'
```

Comment:
I wonder if, to avoid issues in case of the glob'd base path including whitespace, an alternative syntax would make things clearer?  e.g.

```yaml
nav:
  - Intro: 'index.md'
  - Components: '*include ./components/*/mkdocs.yml'
  - With Spaces: '*include ./etc/with spaces/*/mkdocs.yml'
```

That way the instructions on how to use this would be a little simpler too, e.g. "You can provide a glob pattern instead of a single file by using `*include` instead of `!include`"

Reply:
I like this and agree that it would be simpler and more consistent with the current usage of `!include `. I confirmed that the current implementation does break with spaces in the path (https://github.com/backstage/mkdocs-monorepo-plugin/pull/67#discussion_r780278461) so will make sure to address that and add tests while I try out this alternative syntax.

Reply:
I modified the accepted syntax in https://github.com/backstage/mkdocs-monorepo-plugin/pull/67/commits/216091c1b7231ac8f367796d72ed47dd3a3ab49d and changed around the tests. Let me know what you think.